content/guides/guides/command-line.md

```diff
@@ -116,7 +116,7 @@ cypress run [options]
 | `--env`, `-e`              | [Specify environment variables](#cypress-run-env-lt-env-gt)                                                                                                                                |
 | `--group`                  | [Group recorded tests together under a single run](#cypress-run-group-lt-name-gt)                                                                                                          |
 | `--headed`                 | [Displays the browser instead of running headlessly](#cypress-run-headed)                                                                                                                  |
-| `--headless`               | [Hide the browser instead of running headed (default during `cypress run`)](#cypress-run-headless)                                                                                         |
+| `--headless`               | [Hide the browser instead of running headed (default during `cypress run`)](#cypress-run-headless) this option is deprecated                                                               |
```

Comment:
@santoshyadavdev I just double check our CLI options & this option isn't deprecated. Did you come across something that indicated this?

Reply:
Yes actually saw the message from CLI that this option is deprecated.